### PR TITLE
Update attack-simulation-training-faq.md

### PR DESCRIPTION
**Pull request by cammurray**
Added known issue to documentation where customer has set up a custom mailbox in submissions policy but has not followed pre-requisites for that mailbox.

### CODE REVIEW

**Comment by cammurray** [comment]
@dhairyya @stuartcl fyi

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: microsoft-365/security/office-365-security/attack-simulation-training-faq.md [hunk 1]
```diff
@@ -130,6 +130,12 @@ If messages that users reported as phishing aren't captured in Attack simulation
 - phish@office365.microsoft.com
 - not\_junk@office365.microsoft.com
 
+### Users are assigned training after they report a simulated message
+
+If users are being assigned training after they report a phishing simulation message, check to see if your organization has a **custom mailbox** configured in your **User Submission Policy**. When configuring a **custom mailbox** this mailbox needs to be excluded from Safe Links & Safe Attachments policies as per the [Custom mailbox prerequisites](user-submission.md).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/attack-simulation-training-faq.md]:
```suggestion
If users are assigned training after they report a phishing simulation message, check to see if your organization has a **custom mailbox** configured in your **User Submission Policy**. When configuring a **custom mailbox**, this mailbox needs to be excluded from Safe Links & Safe Attachments policies as per the [Custom mailbox prerequisites](user-submission.md).
```

File: microsoft-365/security/office-365-security/attack-simulation-training-faq.md [hunk 2]
```diff
@@ -130,6 +130,12 @@ If messages that users reported as phishing aren't captured in Attack simulation
...
+
+If your organization has a **custom mailbox** configured, and has not set up the required exclusions, these messages may be detonated causing training assignments.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/attack-simulation-training-faq.md]:
```suggestion
If your organization has a **custom mailbox** configured and has not set up the required exclusions, these messages may be detonated, causing training assignments.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @cammurray, kindly commit the suggested changes. Many thanks.

**Pull request merged**